Assets/_BForBoss/_VFX/Scripts/PeriodicallyRestartEffect.cs

```diff
@@ -0,0 +1,31 @@
+//-----------------------------------------------------------------------
+// <copyright file="PeriodicallyRestartEffect.cs" company="PERIGON GAMES">
+//     Copyright (c) PERIGON GAMES. 2020 All Rights Reserved
+// </copyright>
+//-----------------------------------------------------------------------
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.VFX;
+
+namespace BForBoss
+{
+    [ExecuteInEditMode]
+    public class PeriodicallyRestartEffect : MonoBehaviour
+    {
+
+        [SerializeField] private float _waitTime = 1.0f;    
+        [SerializeField] private float _currTime = 0f;      
```

Comment:
any reason why this should be `SerializeField`?

Reply:
Yeah the _currTime probably doesn't need to be.